src/main/java/uk/ac/ebi/ampt2d/metadata/sra/common/TypeBGISEQModel.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
```

Comment:
i guess it would be nice if we move enums to model/enum package. its up to you .